### ISSUE TITLE
Gộp 2 tập dữ liệu

### ISSUE THREAD

**Issue by Trong1102** [Trong1102/Machine_Translation-en-vi#1]
trong readme.md có link của 2 tập dữ liệu, gộp tập train và test của tập dataset1 vào tập dataset2 theo fomat của dataset2. Sau khi làm xong commit code
Deadline: 22h 14/6/2023

**Comment by quantienquanz** [comment]
[https://colab.research.google.com/drive/1pfS1btY_qtw-3VNUJuY0UJK0k7j984I_#scrollTo=wa33uVjTc57D](url)

**Comment by quantienquanz** [comment]
import pandas as pd
import os

dataset1_path = "/content/drive/MyDrive/Colab Notebooks/dataset1"
dataset2_path = "/content/drive/MyDrive/Colab Notebooks/dataset2"

train_dataset1 = pd.read_csv(os.path.join(dataset1_path, "train.csv"))
test_dataset1 = pd.read_csv(os.path.join(dataset1_path, "test.csv"))

train_dataset2 = pd.read_csv(os.path.join(dataset2_path, "train.en"), sep="\t", names=["en"])
test_dataset2 = pd.read_csv(os.path.join(dataset2_path, "tst2012.en"), sep="\t", names=["en"])

train_dataset2 = pd.concat([train_dataset2, train_dataset1["en"], test_dataset1["en"]], axis=0)

train_dataset2.to_csv("/content/drive/MyDrive/Colab Notebooks/dataset3/train.en", sep="\t", index=False, header=False)

test_dataset2 = pd.concat([test_dataset2, test_dataset1["en"]], axis=0)

test_dataset2.to_csv("/content/drive/MyDrive/Colab Notebooks/dataset3/tst2012.en", sep="\t", index=False, header=False)

train_dataset2_vi = pd.read_csv(os.path.join(dataset2_path, "train.vi"), sep="\t", names=["vi"])
test_dataset2_vi = pd.read_csv(os.path.join(dataset2_path, "tst2012.vi"), sep="\t", names=["vi"])

train_dataset2_vi = pd.concat([train_dataset2_vi, train_dataset1["vi"], test_dataset1["vi"]], axis=0)
train_dataset2_vi.to_csv("/content/drive/MyDrive/Colab Notebooks/dataset3/train.vi", sep="\t", index=False, header=False)

test_dataset2_vi = pd.concat([test_dataset2_vi, test_dataset1["vi"]], axis=0)
test_dataset2_vi.to_csv("/content/drive/MyDrive/Colab Notebooks/dataset3/tst2012.vi", sep="\t", index=False, header=False)
